_posts/2015-01-05-organizers.md

```diff
@@ -14,4 +14,4 @@ encoding: utf-8
 * [Ewan Higgs](https://github.com/ehiggs) - [Western Digital](http://www.wdc.com)
 * [Luca Bruno](https://github.com/lucab) - [CoreOS](http://coreos.com/)
```

Comment:
@itkovian would you mind removing my affiliation? My old employer does not exist anymore and I haven't discussed this with the new one.

Reply:
Sure!